core/snippets/snippets/javascript.snippet

```diff
@@ -0,0 +1,56 @@
+language: javascript
+---
+
+name: forInLoopStatement
```

Comment:
this one is duplicate of `forEachStatement`

Reply:
It's not. `of` vs `in`